model_zoo/lbann2.cpp

```diff
@@ -298,13 +298,16 @@ bool load_model_weights(std::string ckpt_dir, model * m){
   sprintf(latest, "%s/last.shared.checkpoint", ckpt_dir.c_str());
```

Comment:
Instead of finding out the last ckpt state under ckpt_dir, would it be beneficial to use ckpt_dir directly. This would give users an option to choose which state to begin with?

Reply:
I have it find the latest checkpoint only because that's the default usage of checkpoint/restart. I see how giving users the option could be beneficial. Do you mean rather than `--ckpt_dir=my_ckpt/` and then deciding which epoch/step to load from instead have a user give `--ckpt_dir/shared.model.0.epoch.#.step.#` and actually specify which checkpoint they would like to load from?

Reply:
yes

Reply:
I think that change makes sense, I will add it.